### ISSUE TITLE
Comments should be less visible

### ISSUE THREAD

**Issue by widersky** [zed-industries/community#2160]
### Check for existing issues

- [X] Completed

### Describe the feature

In most of available themes here the comments are too "insistent". Only in the `One Dark' theme they look good in my opinion, because they are slightly dimmed and do not catch the eye so much.

**Comment by jansol** [comment]
I assume this would be solved by #538

**Comment by widersky** [comment]
@jansol For me "solved" means "OK by default" - not "you can solve it yourself" 😅

**Comment by ojkelly** [comment]
> Only in the `One Dark' theme they look good in my opinion, because they are slightly dimmed and do not catch the eye so much.

Agree. I've been using `Rose Pine Moon` which also has subtler comments.

I'd prefer to be using `Ayu Mirage`, my favourite theme in VSCode, but the comments carry the same weight as the code. In the [original source](https://github.com/dempfi/ayu?tab=readme-ov-file#mirage-with-ui_separator-option-off) for `ayu` they're not as prominent.

Could we get some of the comments toned down, at least until #538 is released?
